layout/_macro/post.swig

```diff
@@ -271,11 +271,7 @@
           {% set COLUMN_NUMBER = 3 %}
           {% for photo in post.photos %}
             {% if loop.index0 % COLUMN_NUMBER === 0 %}<div class="post-gallery-row">{% endif %}
-              <a class="post-gallery-img fancybox"
-                 href="{{ url_for(photo) }}" rel="gallery_{{ post._id }}"
-                 itemscope itemtype="http://schema.org/ImageObject" itemprop="url">
```

Comment:
But why u remove `http://schema.org/ImageObject` here?

Reply:
This attribute belongs to the `<a>` tag which causes the bug, and that's why I removed it. I'll try to add it back, and test if fancybox still works fine.